### DISCUSSION TITLE
Prism Launcher downloading Fabric mods via build in downloader when using Sinytra on forge

### DISCUSSION THREAD

**Discussion by Danieloxor** [Sinytra/Connector#791] [Show and tell]
So, if anyone is using Prism Launcher. you know that the mod browser shows only mods for installed ModLoader. in that case it's Forge, and you have to download fabric mods manually. And i found a workaround for this. 

I noticed that when you install Fabric and Forge in the same instance. Prism shows mods for both loaders, but of course you can't run the game like that. And im not even sure what i did but it worked, i guess i removed all the contents of Fabric Loader except mod id and version. and the Prism still thinks the Fabric is installed and game starts normally because Fabric isn't actually loading

Here is Reddit post i made that explains how to do it and provides some tips to use this method without crashes caused by extra dependencies etc. - [Reddit](https://www.reddit.com/r/PrismLauncher/comments/1aeoj2q/tiphackworkaround_for_downloading_fabric_mods/?utm_source=share&utm_medium=web2x&context=3)

![image](https://github.com/Sinytra/Connector/assets/113946717/f34757cf-f899-4758-b32a-bd4c368caba3)

**Comment by zenfyrdev** [comment]
Prism actually provides a button for that in the menu.

- in the "Version" menu, click "Add Empty"
- In the `uid` field, put `net.fabricmc.fabric-loader` and whatever name you want.
- Click "ok"

The version doesn't matter, as neither Modrinth nor CurseForge check the loader version.

**Reply by Danieloxor** [reply]
Yes, i was trying that. and it didn't worked for me. maybe i screwed something up? idk. 
If adding empty without version number also works, then my method is overcomplicated. But it worked so i was happy.
I wasn't able find any info how to see both fabric and forge mods in downloader so i improvised. I'll try again with adding empty later today, and edit my post if it works.

Thanks for info

**Reply by Danieloxor** [reply]
Yup. you was right. i did it the harder way
